metric_tank/metric_tank.go

```diff
@@ -28,13 +28,15 @@ var (
 	showVersion = flag.Bool("version", false, "print version string")
 	dryRun      = flag.Bool("dry", false, "dry run (disable actually storing into cassandra")
 
-	concurrency = flag.Int("concurrency", 10, "number of workers parsing messages")
-	topic       = flag.String("topic", "metrics", "NSQ topic")
-	channel     = flag.String("channel", "tank", "NSQ channel")
-	instance    = flag.String("instance", "default", "instance, to separate instances in metrics")
-	maxInFlight = flag.Int("max-in-flight", 200, "max number of messages to allow in flight")
-	chunkSpan   = flag.Int("chunkspan", 120, "chunk span in seconds")
-	numChunks   = flag.Int("numchunks", 5, "number of chunks to keep in memory. should be at least 1 more than what's needed to satisfy aggregation rules")
+	concurrency        = flag.Int("concurrency", 10, "number of workers parsing messages")
+	topic              = flag.String("topic", "metrics", "NSQ topic")
+	topicNotifyPersist = flag.String("topic-notify-persist", "metricpersist", "NSQ topic")
+	channel            = flag.String("channel", "tank", "NSQ channel")
+	instance           = flag.String("instance", "default", "instance, to separate instances in metrics")
```

Comment:
separate instance in metrics... and in clustering now, very useful to mention the importance here.